src/RuleAction.php

```diff
@@ -646,17 +646,13 @@ public function displayActionSelectPattern($options = [])
                             $display            = true;
                             break;
 
-                        case "dropdown_validation_percent":
-                            $ticket = new Ticket();
-                            echo $ticket->getValueToSelect('validation_percent', 'value', $param['value']);
-                            $display       = true;
-                            break;
-
-                        case "dropdown_validation_status":
-                            TicketValidation::dropdownStatus('value', [
-                                'global' => true,
-                                'value' => $param['value'],
-                            ]);
+                        case "dropdown_percent":
+                            $percentages = range(0, 100, 10); // Liste des pourcentages par pas de 10
```

Comment:
```suggestion
                            $percentages = range(0, 100, 10); // List of percentages in increments of 10
```

Reply:
Do we really want to limit validation percentages to intervals of 10? It makes cases like 66% or 75% support impossible.

Reply:
I have no strong opinion about this.
If someone want to have 1 validation on 3, he has to set 30% and that will do the job. So there is no real usage limitation by this.
Feedback from users will help us to decide what to do.